corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js

```diff
@@ -110,6 +114,11 @@ hqDefine("cloudcare/js/formplayer/menus/collections", [
                 _.pick(response, ["queryKey", "selections"]),
                 _.identity
             );
+            if (response.locales && !UsersModels.getCurrentUser().displayOptions.singleAppMode
```

Comment:
1. ohh good catch - that was from when I was using this conditional to call `startForm`, which would break app preview's ability to render case lists at all. 
2. and not really! I simplified the conditional so that any response that isn't a form will have the language menu available